vaadin-platform-react-hybrid-security-test/src/main/java/com/vaadin/platform/react/test/security/SecurityConfig.java

```diff
@@ -26,7 +26,7 @@ public SecurityConfig(RouteUtil routeUtil) {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeHttpRequests(registry -> {
-            registry.requestMatchers(new AntPathRequestMatcher("/")).permitAll();
+            registry.requestMatchers(PathPatternRequestMatcher.pathPattern("/")).permitAll();
```

Comment:
IIRC, with SB4 it should be fine to just pass the pattern.

```suggestion
            registry.requestMatchers("/").permitAll();
```